py/sentry_streams/user_functions/sample_agg.py

```diff
@@ -0,0 +1,39 @@
+from typing import MutableMapping
+
+from sentry_streams.user_functions.function_template import (
+    Accumulator,
+    AggregationBackend,
+)
+
+WordCountTuple = tuple[str, int]
+WordCountStr = str
```

Comment:
Inline these ? You do not need to declare the type when instantiating a class that is generic. 
Defining the type aliases is generally useful when the name adds something semantically to the bare types. Here it does not seem it is happening as the name of the alias is basically the name of the bare types.